travis/setup_dependencies_common.sh

```diff
@@ -9,6 +9,10 @@ conda config --add channels defaults
 
 shopt -s nocasematch
 
+LATEST_ASTROPY_STABLE=1.1
+LATEST_ASTROPY_LTS=1.0
+LATEST_NUMPY_STABLE=1.11
```

Comment:
You're right with 1.5 vs latest stable, but we do the same with when
installing the development or the prerelease version.

On 20 June 2016 at 11:37, Thomas Robitaille <notifications@github.com>
wrote:

> In travis/setup_dependencies_common.sh
> <https://github.com/astropy/ci-helpers/pull/103#discussion_r67668740>:
>
> > @@ -9,6 +9,10 @@ conda config --add channels defaults
> >
> >  shopt -s nocasematch
> >
> > +LATEST_ASTROPY_STABLE=1.1
> > +LATEST_ASTROPY_LTS=1.0
> > +LATEST_NUMPY_STABLE=1.11
>
> I don't think things will work well with Numpy, because if you update just
> numpy from 1.5 to 1.11, I think a lot of packages built on Numpy will fail.
> I really hope the conda build gets fixed soon.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/astropy/ci-helpers/pull/103/files/5745166c2395eb9e6951fa79c872f01d6e8eefd3#r67668740>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGeUwtayfJ00B0Cy8qCG8kkTThBZcPaMks5qNm1ugaJpZM4I5jvO>
> .
>